### PR TITLE
EKA-876 | Gayatri/Sangita | Refactor code the for consentStatus Granted

### PR DESCRIPTION
**Pull request by avghadsangita**

### CODE REVIEW

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Clones added
============
- src/In.ProjectEKA.HipService/Consent/ConsentNotificationController.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6110512&bid=19999617)

**Review by Nexengineer — APPROVED** [review]

File: src/In.ProjectEKA.HipService/Consent/ConsentNotificationController.cs [hunk 1]
```diff
@@ -51,6 +51,14 @@ public async Task StoreConsent(ConsentArtefactRepresentation consentArtefact)
                     notification.Status,
                     notification.ConsentId);
                 await consentRepository.AddAsync(consent);
+                var cmSuffix = consent.ConsentArtefact.ConsentManager.Id;
+                var gatewayResponse = new GatewayRevokedConsentRepresentation(
```
Comment (mddubey) [inline src/In.ProjectEKA.HipService/Consent/ConsentNotificationController.cs:55]: Why `GatewayRevokedConsentRepresentation`? This is not the case of revoke.

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/Gateway/Model/GatewayConsentRepresentation.cs  1
         

Clones added
============
- src/In.ProjectEKA.HipService/Consent/ConsentNotificationController.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=6110512&bid=19999617)

File: src/In.ProjectEKA.HipService/Consent/ConsentNotificationController.cs (hunk 1 above)
Comment (gayatrijagtap) [reply src/In.ProjectEKA.HipService/Consent/ConsentNotificationController.cs]: renamed

**Pull request merged**